tests/python/pants_test/engine/legacy/test_console_rule_integration.py

```diff
@@ -13,29 +13,19 @@
 
 class TestConsoleRuleIntegration(PantsDaemonIntegrationTestBase):
 
+  list_target = 'examples/src/java/org/pantsbuild/example/hello::'
+
   @ensure_daemon
   def test_v2_list(self):
-    result = self.do_command(
-      '--no-v1',
-      '--v2',
-      'list',
-      '::',
-      success=True
-    )
-
+    result = self.do_command('list', self.list_target, success=True)
     output_lines = result.stdout_data.splitlines()
-    self.assertGreater(len(output_lines), 1000)
-    self.assertIn('3rdparty/python:psutil', output_lines)
+    self.assertEqual(len(output_lines), 5)
+    self.assertIn('examples/src/java/org/pantsbuild/example/hello/main:readme', output_lines)
 
   def test_v2_list_does_not_cache(self):
     with self.pantsd_successful_run_context() as (runner, checker, workdir, pantsd_config):
       def run_list():
-        command = ['--no-v1',
-                   '--v2',
-                   'list',
-                   'src/::',]
-
-        result = runner(command)
+        result = runner(['list', self.list_target])
         checker.assert_started()
         return result
```

Comment:
I'm not clear how this test really is checking that we don't cache. A sanity check on this change (i.e. not running against all of `src/::` would be helpful).

Reply:
I know this is merged, but just in case.
There was a problematic interaction between pantsd and the v2 graph a while ago.
The `@console_rules` would cache their result in the v2 graph, so two consecutive runs of a console_rule under pantsd would only run once, and the output wouldn't show on the second run.

Reply:
Got it. So this test should still work for checking that, right?

Also note that this does not have `@ensure_daemon`. Is that intentional?

Reply:
> Got it. So this test should still work for checking that, right?

Yeap, it's a regression test, to make sure it never happens again.

> Also note that this does not have @ensure_daemon. Is that intentional?

Yeah, the context manager is set to `pantsd_successful_run`, which will only trigger under pantsd. The nature of the bug that this regression test is testing means that pantsd is the only place it will manifest.